tsuru/client/plan.go

```diff
@@ -201,7 +206,11 @@ func renderPlansK8SFriendly(plans []apptypes.Plan, showMaxBurstAllowed bool) str
 	for _, p := range plans {
 		memory := resource.NewQuantity(p.Memory, resource.BinarySI).String()
 		cpuRequest := resource.NewMilliQuantity(int64(p.CPUMilli), resource.DecimalSI).String()
-		maxCPULimit := resource.NewMilliQuantity(int64(float64(p.CPUMilli)*p.CPUBurst.MaxAllowed), resource.DecimalSI).String()
+		maxAllowed := 0.0
+		if p.CPUBurst != nil {
+			maxAllowed = p.CPUBurst.MaxAllowed
+		}
+		maxCPULimit := resource.NewMilliQuantity(int64(float64(p.CPUMilli)*maxAllowed), resource.DecimalSI).String()
```

Comment:
@wpjunior I'm not sure if these changes are correct. Basically in tsuru latest `Plan.CPUBurst` and `Plan.Override` are now pointers so we have to check for a nil dereference. All the fixes here were panicking the tests, I will do a code search for the use of these types and update accordingly.

Reply:
@mauri870 just a adjustment: maxAllowed default is 1.0, just this adjustment to allow this MR.

Reply:
Thanks, I fixed it. I'm pretty sure there are more cases where we can get panics if we do not check for nil. If you feel like this is fine you can merge this as-is and fix these later. I'd advise to search for the uses of Override and CPUBurst and add the nil checks as needed.